app/auto-eligibility/register-your-interest/process-duplicate-email.js

```diff
@@ -0,0 +1,28 @@
+const emailNotifier = require('../email-notifier')
+const raiseTelemetryEvent = require('../telemetry/raise-telemetry-event')
+const telemetryEvent = require('../telemetry/telemetry-event')
+
+const processDuplicateEmail = async (businessEmail) => {
+  console.log(`${new Date().toISOString()} Processing duplicate business email: ${JSON.stringify({
+    businessEmail
+  })}`)
+  await emailNotifier.sendIneligibleDefraIdRegistrationEmail(businessEmail)
+  await raiseTelemetryEvent(businessEmail)( // TODO: make this raise a proper telemetry event
```

Comment:
As @marcin-mogiela has already mentioned, this may not be required as we do not need to log events for MI reporting for the new register your interest journey